resources/assets/coffee/react/mp-history/header.coffee

```diff
@@ -0,0 +1,28 @@
+###
+# Copyright 2016 ppy Pty. Ltd.
+#
+# This file is part of osu!web. osu!web is distributed with the hope of
+# attracting more community contributions to the core ecosystem of osu!.
+#
+# osu!web is free software: you can redistribute it and/or modify
+# it under the terms of the Affero GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+{div, h1, h2} = React.DOM
+
+class MPHistory.Header extends React.Component
```

Comment:
I don't think it's used anywhere else yet, but for this kind of component, [stateless function](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions) can be used.